modules/serverless-kafka-sasl-channels.adoc

```diff
@@ -1,23 +1,26 @@
 // Module included in the following assemblies:
 //
-// * serverless/security/serverless-kafka-security.adoc
+// * serverless/admin_guide/serverless-kafka-admin.adoc
 
 :_content-type: PROCEDURE
-[id="serverless-kafka-sasl_{context}"]
-= Configuring SASL authentication
+[id="serverless-kafka-sasl-channels_{context}"]
+= Configuring SASL authentication for Kafka channels
 
-You can use the following procedure to configure SASL authentication for a Kafka channel.
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed. You can set up SASL for Kafka channels by modifying the `KnativeKafka` custom resource (CR).
```

Comment:
The CR (for channel) just points to the secret, which the customer must itself. We don't config the actual details of the secret on the CR, though

Reply:
Yeah, I think I'll remove the line about the CR from the modules tbh